src/fabric/gamma/SplitViewHostNativeComponent.ts

```diff
@@ -43,6 +43,8 @@ type SplitViewOrientation =
   | 'landscapeLeft'
   | 'landscapeRight';
 
+type SplitViewPrimaryBackgroundStyle = 'systemDefault' | 'none' | 'sidebar';
```

Comment:
```suggestion
type SplitViewPrimaryBackgroundStyle = 'default' | 'none' | 'sidebar';
```

I'd just name it `default` or `auto`. I'm skeptical sceptical of `systemDefault` because well, it's not "system" default, it's UIKit default. I'm being very nitpicky here. 

I think `default` is the best here, as it regards the styling. `auto` would be better for behaviour related prop. 

We need to look into our naming schemes & create some guidelines. I'll take care of it soon.